.github/runlint.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+###############################################################################
+# Copyright (c) 2021, 2021 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+# Exit immediately if any unexpected error occurs.
+set -e
+# trace the commands that are run
+set -x
+
+# github runner ubuntu-16.04 has 2 cores and 8 gigs of memory.  Attempt to double provision the number of cores for the make.
+MAKE_JOBS=4
+
+# explicitly select the desired version of GCC via the environment
+export CC=gcc-7
+export CXX=g++-7
+
+# download bootstrap jdk
+SRCDIR=$PWD
+cd ~
+export JAVA_HOME="$JAVA_HOME_11_X64"
+export PATH="$JAVA_HOME/bin:$PATH"
+cd $SRCDIR
```

Comment:
The comment is now stale: it can simply be removed.
`SRCDIR` is no longer necessary, because the effect of the two `cd` commands is to go nowhere.